src/client/containers/ProfilePage/StudentProfile/StudentProfile/StudentProfile.style.css

```diff
@@ -0,0 +1,71 @@
+.student-headerProfilePage {
+  width: 100%;
+  height: fit-content;
+  border-bottom-left-radius: 10em;
+  border-bottom-right-radius: 10em;
+  background-color: #f8f8f8;
+}
+
+#Vector {
```

Comment:
Why did you use ids for these images instead of classes?
Also, it would be a good practice to prefix all selectors in this file with `.student-profile-page ` in case something like `#Vector` is styled also in another file, but differently.